aqt/cli.py

```diff
@@ -83,6 +83,8 @@ def _set_arch(self, args, oarch, os_name, target):
                 arch = "clang_64"
             elif os_name == "mac" and target == "ios":
                 arch = "ios"
+            elif target == "android":  # TODO only works for Qt >= 5.14. Somehow parse the version?
```

Comment:
As you can see, we theoretically have to check if `qt_version` is >= 5.14 to prevent user errors. However, mentioning this in the README might be enough. What do you prefer:

- Using a version comparison like: https://packaging.pypa.io/en/latest/version/#packaging.version.parse
- By adding a hint to the readme

Reply:
I prefer both. 

When using version comparison and set default values.
When 5.14.0 and later, set 'android' and when 5.13.x and prior set 'android_armv7'.
If a default value is natural, we can set default. If not, it should be explicitly specified as same as MSVC arch parameter.